x-pack/plugins/remote_clusters/public/store/actions/remove_clusters.js

```diff
@@ -75,7 +74,6 @@ export const removeClusters = (names) => async (dispatch, getState) => {
 
   dispatch({
     type: REMOVE_CLUSTERS_FINISH,
+    payload: removalSuccesses,
```

Comment:
In the future I think it would be good practice for us to either a) list changes like these in the PR description or (best option) b) capture them in a commit with a descriptive message (second best). That will make it easier to understand how and why the code changed both during review and if we ever need to look through past PRs and piece together the evolution of this part of the codebase. 😄

Reply:
Good point! I should have explained it in the description. I will update it now.